base/base/wide_integer_impl.h

```diff
@@ -281,6 +288,14 @@ struct integer<Bits, Signed>::_impl
         }
     }
 
+    constexpr static void wide_integer_from_cityhash_uint128(integer<Bits, Signed> & self, const CityHash_v1_0_2::uint128 & value) noexcept
```

Comment:
`CityHash_v1_0_2::uint128` is not tuple-like anymore so I had to add this function to implement conversion `CityHash_v1_0_2::uint128` -> `UInt128`.